spec/factories/user.rb

```diff
@@ -15,6 +15,10 @@
     factory :admin do
       is_admin true
     end
+
+    factory :non_admin do
+      is_admin false
```

Comment:
A basic user when it is not compared to an admin one (i.e., when the spec does not test the app behaviour regarding admin special access)
That was our thinking anyway :) But I see what you mean by keeping the factory light. And it's still comprehensible, we probably overthought this